docs/tutorials/python-development-environment.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Setting up a Python development environment
+subject: Software
+---
+By Anton Vanhoucke
```

Comment:
@dlech How do we want to handle tutorial authorship and attribution? I don't think that we should have people's names inserted manually in the tutorial content, personally. If the author wants attribution, that's reasonable though -- maybe a built-in byline similar to copyright info in code files?